src/views/Pools/components/PoolCard/CardFooter/ExpandedFooter.tsx

```diff
@@ -128,7 +128,7 @@ const ExpandedFooter: React.FC<ExpandedFooterProps> = ({
         </Flex>
       )}
       {account && isMetaMaskInScope && tokenAddress && (
-        <Flex justifyContent="flex-end">
+        <Flex justifyContent="flex-end" style={{ cursor: 'pointer' }}>
```

Comment:
Can we also add underline when hover ?

Reply:
Nice catch, I've added those styles in cheers Memoyil.